packages/sdk/src/types/attributes/requests/CreateRelationshipAttributeRequest.ts

```diff
@@ -0,0 +1,19 @@
+import { ConnectorAttributeValue } from "../ConnectorAttribute";
+
+export interface CreateRelationshipAttributeRequest {
+    content: {
+        value: ConnectorAttributeValue;
```

Comment:
Don't we have a ConnectorIdentityAttributeValue?

Reply:
There exists `ConnectorRelationshipAttribute`, which seems to fit here.